chain/consensus/hierarchical/actors/sca/sca_apply.go

```diff
@@ -41,6 +52,67 @@ func applyFund(rt runtime.Runtime, msg types.Message) {
 	}
 }
 
+func applyRelease(rt runtime.Runtime, msg types.Message) {
```

Comment:
@Lefkok, if you could have a "sanity-check look" to this piece of code I would really appreciate it. This function applies `release` msgs (i.e. bottom-up cross-messages that release funds from a subnet to the parent chain).

(The PR is huge, so don't worry about reviewing it fully, many are just refactors and cosmetic changes).

Thanks